packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -2340,9 +2354,11 @@ export class GetAssetHolding extends Op {
 		}
 		let value: StackElem;
 		switch (this.field) {
+			case "0":
```

Comment:
case "0"? what does it do?

Reply:
case "0" is the index for `AssetBalance`.